app/assets/stylesheets/application/common.scss

```diff
@@ -216,3 +216,8 @@ div.centered {
     &+input {display: none;}
   }
 }
+.red-text {
+  color: #CE2D29;
+  font-size: 15px;
+
+}
```

Comment:
It is displayed in red. But there seems indentation problem. If u can merge can correct it ^_^